docs/announcements/en/2018-12-18-registering-pickup-points-is-now-easier-and-faster.md

```diff
@@ -15,7 +15,7 @@ announcementImageID: ''
 announcementSynopsisEN: 'Now, you can upload an XLS file containing all the pickup points you want to offer on your website.'
 ---
 
-You don't need to register pickup points one by one anymore. Now, you can use your Admin to upload an XLS file [(like this one)](https://assets.ctfassets.net/alneenqid6w5/2QZFVRKdaoOkykE4AUkw6m/bdf62b6d028819b5b8c08ef7a27eadc6/pickups_example.xls) with all the pickup points you want to offer on your website.
+You don't need to register pickup points one by one anymore. Now, you can use your Admin to upload an XLS file [(like this one)](//assets.ctfassets.net/alneenqid6w5/2QZFVRKdaoOkykE4AUkw6m/bdf62b6d028819b5b8c08ef7a27eadc6/pickups_example.xls) with all the pickup points you want to offer on your website.
```

Comment:
Image URLs must contain `https://`.

Reply:
Good catch, I had fixed most images, but not assets. I've added 2 commits to fix that now